src/testdrive/junit.py

```diff
@@ -117,7 +151,13 @@ def junit(suite, cases, hostname=None, prettify=False):
         time - test duration in seconds
 
     If `timestamp` is supplied then `time` must also be supplied.
+
+    If both `baseurl_ids` and `baseurl_specs` are supplied then add a property
+    for 'test_specification' by mapping a URL under `baseurl_specs`.
```

Comment:
rephrase this to make it clearer that it's essentially a find+replace to get the spec url.

Reply:
Rephrased